### PR TITLE
Add bootstrap to the frontend

### PR DESCRIPTION
**Pull request by davidemarcoli**

### CODE REVIEW

**Review by Sashimi4 — DISMISSED** [review]
LGTM

File: backend/pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
         <maven.compiler.target>17</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <frontend-bundle-dir>../frontend/dist/project-template/</frontend-bundle-dir>
-        <start-class>de.davidemarcoli.backend.BackendApplication</start-class>
+        <start-class>com.social.dialylink.BackendApplication</start-class>
```
Comment (lazarpe) [inline backend/pom.xml]: com.social.dialylink to com.social.dailylink

**Review by lazarpe — CHANGES_REQUESTED** [review]

File: backend/pom.xml (hunk 1 above)
Comment (davidemarcoli) [reply backend/pom.xml]: Fixed in following commit

**Review by Sashimi4 — APPROVED** [review]
LGTM

**Review by lazarpe — APPROVED** [review]

**Pull request merged**